tool/dart_site/analysis_options.yaml

```
@@ -1 +1,5 @@
 include: package:analysis_defaults/analysis.yaml
+
+linter:
+  rules:
+    - omit_local_variable_types
```

Thanks for adding this by the way!

```suggestion
    - omit_local_variable_types

```